src/components/ChannelPreviewLastMessage.js

```diff
@@ -50,7 +56,10 @@ export class ChannelPreviewLastMessage extends PureComponent {
             <span className="str-chat__channel-preview-last-message">
               {!this.props.channel.state.messages[0]
                 ? 'Nothing yet...'
-                : this.props.latestMessage}
+                : truncate(
```

Comment:
@mahboubii Does this work? Because as per sytax, 2nd parameter is an object for `truncate` function - https://lodash.com/docs/4.17.15#truncate

Reply:
yea last time i manually tested it